mkosi/__init__.py

```diff
@@ -2255,3 +2260,13 @@ def run_verb(args: MkosiArgs, presets: Sequence[MkosiConfig]) -> None:
 
             if args.verb == Verb.serve:
                 run_serve(last)
+
+
+def normalize_mtime(root: Path, directory: Path, mtime: Optional[int]) -> None:
```

Comment:
Please make `directory` optional and default to `/` if it is `None`

Reply:
I think this requires reordering the args again. Should we put mtime first?

Reply:
I would always pass it by keyword